helm-chart/renku/templates/configmap.yaml

```diff
@@ -146,3 +148,22 @@ data:
         grant all privileges on schema "public" to "{{ .Values.global.graph.tokenRepository.postgresUser }}";
       EOSQL
   {{- end }}
+
+  # DO NOT USE set -x you will leak the password in the logs
+  init-renku-db.sh: |-
+    #!/bin/bash
+
+    PASSWORD=$(cat /db-common-password/password)
+    DB_NAME=renku
+
+    psql -v ON_ERROR_STOP=1 <<-EOSQL
+      create database "$DB_NAME";
```

Comment:
We think we've been doing this wrong; we think that we should be creating the db such that the db owner is the new user; hence the user should be created first and the db should be created with this user as owner.

Reply:
I changed this as you requested. I looked at the other PR where you fixed this for the kg.